lib/auth/oauth/oauth.ts

```diff
@@ -116,9 +136,18 @@ export function callback(config: ConfigInterface) {
     ).searchParams;
     const shop = query.get('shop')!;
 
+    const response = {} as NormalizedResponse;
+    if (isbot(request.headers['User-Agent'])) {
+      logForBot({request, log, func: 'callback'});
+      return {
+        headers: (await abstractConvertHeaders({}, adapterArgs)) as T,
+        session: {} as Session,
+      };
+    }
```

Comment:
can we not return the same escape as `begin` here and return a `418`?

Reply:
We cannot, we don't handle the response in the API for the callback the same way, instead we're throwing now